src/tests/test_method.py

```diff
@@ -304,6 +304,10 @@ def testValueParamsArgs(self):
         self.assertTrue(result[1] == 2)
         self.assertTrue(result[2] == 3)        
 
+    def testNonParamsArrayInLastPlace(self):
```

Comment:
I mean don't you need to add in c# and check in python tests for the
following case below?

public static bool TestNonParamsArrayInLastPlace(int i1, params int[] ints)
{ return false; }

Maybe boolean return value needs to be changed to integer, since then there
are 3 overloaded cases.

On Wed, Apr 6, 2016 at 10:47 AM, omnicognate <notifications@github.com>
wrote:

> In src/tests/test_method.py
> <https://github.com/pythonnet/pythonnet/pull/201#discussion_r58729780>:
>
> > @@ -304,6 +304,10 @@ def testValueParamsArgs(self):
> >          self.assertTrue(result[1] == 2)
> >          self.assertTrue(result[2] == 3)
> >
> > +    def testNonParamsArrayInLastPlace(self):
>
> Do you mean the case where there is a params keyword. If so, I believe
> that's covered by existing tests, eg. testObjectParamsArgs
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonnet/pythonnet/pull/201/files/4117162e2e98a0d2458c067ac7ea4b69e9a5d8e2#r58729780>
>